File: Python Programming/problems on string manipulation/problem114.py
'''
114. Write a program which accept string from user and copy small characters of that string into another string. 

	Input :   “Marvellous multi OS” 
	Output :   “arvellous multi” 
	
	Author : Milind Nalawade
	
	Date : 16/07/2021
		
	Language : Python 3.9.0
	
'''
#====
#Logic
#====
def CapString(str):
	dest=""
	for i in str:
		if i.islower() or i==" ":
			dest=dest+i
	print("\n Output : {}\n".format(dest))	

#=============
#entry point function
#=============
def main():
	str=input("Enter your string : ")
	CapString(str)

#=====
#starter
#=====
if __name__=="__main__":
	main()

File: C Programming/problems on strings maipulation/problem114.c
/*
114. Write a program which accept string from user and copy small characters of that string into another string. 

	Input :   “Marvellous multi OS” 
	Output :   “arvellous multi” 

	Author : Milind Nalawade
	
	Date : 16/07/2021
		
	Language : C progrmming 
	
*/	

#include<stdio.h>
#include<string.h>

//Logic
void CopyCap(char* arr,char* brr)
{
	
	while(*arr!='\0')
	{
		if (*arr>='a'&&*arr<='z'||*arr==' ')
		{
			*brr=*arr;
			brr++;	
		}
		arr++;
	}
	*brr='\0';
}

//Entry point function
int main()	
{
	char arr[30];
	char brr[30];
	printf("\n Enter your string : ");
	scanf("%[^'\n']s",&arr);
	CopyCap(arr,brr);
	printf("\n Output : %s\n",brr);
	return 0;
}